src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -193,6 +193,10 @@ public static void UpdateBackground(this AView platformView, Paint? background)
 						platformView.Background = drawable;
 				}
 			}
+			else if(platformView is LayoutViewGroup)
+			{
+				platformView.Background = null;
```

Comment:
Seeing these changes in Android and iOS, is something similar necessary in Windows?